app/components/primer/alpha/select_panel.rb

```diff
@@ -429,6 +409,13 @@ def initialize(
         @dynamic_aria_label_prefix = dynamic_aria_label_prefix
         @loading_label = loading_label
         @loading_description_id = nil
+
+        @form_builder = form_arguments[:builder] if remote_form_arguments
+        @value = form_arguments[:value] if remote_form_arguments
+        @input_name = form_arguments[:name] if remote_form_arguments
+
+        @list_form_arguments = remote_form_arguments ? {} : form_arguments
```

Comment:
I think we can improve readability here by nesting under one conditional!

```
if remote_form_arguments
   ...
else
  @list_form_arguments = form_arguments
end
```

Reply:
I need the tertiary for the `list_form_arguments` but I can do the if for the other args.